### PR TITLE
[2X branch]: Migrate dockerhub images to gpc

### PR DESCRIPTION
**Pull request by dgkimura**
- Update to pull images from GCP.
    https://github.com/pivotal/gp-image-baking#supported-images
- Replace docker-image with registry-image.
    find . -type f -exec sed -i -e 's/docker-image/registry-image/g' {} \;
    
For some reason, seems `pivotaldata/centos-gpdb-dev:6` wasn't moved into `gcr.io/data-gpdb-public-images`.  I guess we're the only one who use it so moved into `gcr.io/data-orca/centos-gpdb-dev:6`.

### CODE REVIEW

File: concourse/test_orca_2x_pipeline.yml [hunk 1]
```diff
@@ -97,14 +97,14 @@ resources:
...
-    tag: "5"
+    repository: gcr.io/data-gpdb-public-images/gpdb4-centos5-build-test
+    tag: latest
 - name: centos-gpdb-dev-6
-  type: docker-image
+  type: registry-image
   source:
-    repository: pivotaldata/centos-gpdb-dev
+    repository: gcr.io/data-orca/centos-gpdb-dev
     tag: "6"
```
Comment (chrishajas) [inline concourse/test_orca_2x_pipeline.yml:108]: Is using the "6" tag ok here?

**Review by chrishajas — APPROVED** [review]
LGTM (also needs patch bump)

File: concourse/test_orca_2x_pipeline.yml (hunk 1 above)
Comment (dgkimura) [reply concourse/test_orca_2x_pipeline.yml:108]: Yeah, when copying it over to data-orca I kept the same tag.

```
$ gcloud container images list-tags gcr.io/data-orca/centos-gpdb-dev
DIGEST        TAGS  TIMESTAMP
8b3b90472b66  6     2021-05-27T23:19:01
```

**Comment by dgkimura** [comment]
Bumped patch version and merged. Thanks for review!

**Pull request merged**